frontend/src/components/organization/OrganizationPreviewBody.js

```diff
@@ -0,0 +1,92 @@
+import { Box, CardContent, Tooltip, Typography } from "@material-ui/core";
+import React, { useState } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import PlaceIcon from "@material-ui/icons/Place";
+
+const useStyles = makeStyles((theme) => {
+  return {
+    bold: {
+      fontWeight: "bold",
+    },
+    locationName: {
+      fontWeight: "bold",
+      whiteSpace: "nowrap",
+      width: "100%",
+      overflow: "hidden",
+      OTextOverflow: "ellipsis",
+      textOverflow: "ellipsis",
+    },
+    infoLink: {
+      display: "flex",
+    },
+    cardIconBox: {
+      width: 40,
+      flex: "0 0 40px",
+      display: "inline-block",
+    },
+    textContent: {
+      fontSize: 14,
+      whiteSpace: "normal",
+    },
+    locationBox: {
+      display: "grid",
+      gridTemplateColumns: "40px min-content",
+      marginBottom: theme.spacing(0.5),
+      justifyContent: "center",
+    },
+    locationNameBox: {
+      maxWidth: "220px",
+      overflow: "hidden",
+    },
+    shortenedSummary: {
+      overflow: "hidden",
+      WebkitBoxOrient: "vertical",
+      display: "-webkit-box",
+    },
+    summaryBox: {
+      overflow: "hidden",
+    },
+    contentWrapper: {
+      padding: 0,
+      display: "grid",
+      gridTemplateRows: "min-content",
+    },
+  };
+});
+
+export default function OrganizationPreviewBody({ organization }) {
+  const classes = useStyles();
+  const [linesOfText, setLinesOfText] = useState(5);
+
+  return (
+    <CardContent className={classes.contentWrapper}>
+      <Box>
+        {!!organization.info.location && (
+          <span className={classes.locationBox}>
+            <span className={classes.cardIconBox}>
+              <Tooltip title="location">
+                <PlaceIcon className={classes.placeIcon} color="primary" />
+              </Tooltip>
+            </span>
+            <span className={classes.locationNameBox}>
+              <Typography variant="subtitle1" component="h3" className={classes.locationName}>
+                {organization.info.location}
+              </Typography>
+            </span>
+          </span>
+        )}
+      </Box>
+      <Box
+        className={classes.summaryBox}
+        ref={(textBox) => {
+          const linesEstimation = textBox?.clientHeight / 25;
+          setLinesOfText(Math.floor(linesEstimation));
+        }}
+      >
+        <Typography className={classes.shortenedSummary} style={{ WebkitLineClamp: linesOfText }}>
+          {organization.short_description ?? organization.description}
```

Comment:
Nice, I actually didn't know about the `??` operator. Will use that in the future as well.